modules/raft/src/main/java/org/apache/ignite/raft/jraft/ChangePeersAsyncStatus.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.raft.jraft;
+
+public enum ChangePeersAsyncStatus {
+    DONE(),
+
+    WRONG_TERM(),
+
+    RECEIVED(),
+
+    BUSY();
```

Comment:
Can't find where we are using this status. As I see, the busy status will be reported by jraft closure with an error response. We need to receive this response as other statuses from this PR - in changePeersAsync response